php/libraries/File_Upload.class.inc

```diff
@@ -24,25 +25,31 @@
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris
  */
+
 class File_Upload
 {
+    private $_baseUploadDirectory;
+    private $_destinationDirectory;
+    private $_destinationFilename;
+    private $_errorLog;
+
     /**
      * Construct the File_Upload class. This sets the default parameters
      * for a file which can be overwritten by calling the set* methods
      */
-    function __construct()
+    public function __construct()
     {
-        //Set Defaults
-        $this->renameMode        = "incremental";
```

Comment:
@MounaSafiHarab to your knowledge does anyone have this value set to another option then incremental ??